src/components/chat/chat-messages.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import './chat.scss';
```

Comment:
The chat.scss needs all html element stylings changed ie `form input {width:80%; height:30px }` should be a class name...this will effect all our other elements @zsli16 @nataliaero